src/Aspire.Hosting/Dcp/ResourceLogSource.cs

```diff
@@ -24,25 +24,42 @@ public async IAsyncEnumerator<LogEntryList> GetAsyncEnumerator(CancellationToken
             throw new ArgumentException("Cancellation token must be cancellable in order to prevent leaking resources.", nameof(cancellationToken));
         }
 
-        var timestamps = resource is Container; // Timestamps are available only for Containers as of Aspire P5.
-        var stdoutStream = await kubernetesService.GetLogStreamAsync(resource, Logs.StreamTypeStdOut, follow: true, timestamps: timestamps, cancellationToken).ConfigureAwait(false);
-        var stderrStream = await kubernetesService.GetLogStreamAsync(resource, Logs.StreamTypeStdErr, follow: true, timestamps: timestamps, cancellationToken).ConfigureAwait(false);
-
         var channel = Channel.CreateUnbounded<LogEntry>(new UnboundedChannelOptions
         {
             AllowSynchronousContinuations = false,
             SingleReader = true,
             SingleWriter = false
         });
 
+        var streamTasks = new List<Task>();
+
+        var timestamps = resource is Container; // Timestamps are available only for Containers as of Aspire P5.
+
+        if (resource is Container)
+        {
+            var startupStderrStream = await kubernetesService.GetLogStreamAsync(resource, Logs.StreamTypeStartupStdErr, follow: true, timestamps: timestamps, cancellationToken).ConfigureAwait(false);
+            var startupStdoutStream = await kubernetesService.GetLogStreamAsync(resource, Logs.StreamTypeStartupStdOut, follow: true, timestamps: timestamps, cancellationToken).ConfigureAwait(false);
+
+            var startupStdoutStreamTask = Task.Run(() => StreamLogsAsync(startupStdoutStream, isError: false), cancellationToken);
+            streamTasks.Add(startupStdoutStreamTask);
+
+            var startupStderrStreamTask = Task.Run(() => StreamLogsAsync(startupStderrStream, isError: false), cancellationToken);
```

Comment:
Why is this using `isError: false`?

Reply:
Deliberate. `docker build` outputs vritually everything to stderr which @karolz-ms points out is the right thing for it to do from a unix philosophy point of view. But given it prepends the `err` text to ever line its quite noisy.

I think we might want to have a chat to the dashboard folks about a slightly different visual treatment here.